src/data/matter.ts

```diff
@@ -1,4 +1,70 @@
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import { navigate } from "../common/navigate";
+import { showAlertDialog } from "../dialogs/generic/show-dialog-box";
 import { HomeAssistant } from "../types";
+import { subscribeDeviceRegistry } from "./device_registry";
+
+export const canCommissionMatterExternal = (hass: HomeAssistant) =>
+  hass.auth.external?.config.canCommissionMatter;
+
+export const startExternalCommissioning = (hass: HomeAssistant) =>
+  hass.auth.external?.fireMessage({
+    type: "matter/commission",
+  });
+
+export const addMatterDevice = (element: HTMLElement, hass: HomeAssistant) => {
+  if (!canCommissionMatterExternal(hass)) {
+    showAlertDialog(element, {
+      title: "Use mobile app",
+      text: "Matter commissioning is not supported on this device, use the mobile app to commission Matter devices",
+    });
+    return;
+  }
```

Comment:
Let's also move this to it's own function, so we have more control to not even call `addMatterDevice` and know earlier why it failed.

Reply:
`canCommissionMatterExternal` already is it's own function, that can be used if we want more control, but didn't want to add that everywhere

Reply:
alright let's keep that one.